### PR TITLE
Pr/45

### PR DESCRIPTION
**Pull request by NoahIles**

### CODE REVIEW

File: app/realDealz/library.py [hunk 1]
```diff
@@ -104,14 +104,20 @@ def m_get(self, url : str, p_headers : dict):
...
         }
-        _headers.update(self.a_header)
         if p_headers is not None:
             _headers.update(p_headers)
-        result = json.loads(http.request(
-            'GET',
-            url,
-            headers=_headers
-        ).data)
+
```
Comment (nickhib) [inline app/realDealz/library.py:109]: does this inherently change anything for the use of the library API i don't think so but i was just confused would this be a better method for get request?

File: app/requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ Django==4.1.7
 environs==7.3.1
 urllib3==1.26.14
 setuptools==67.4.0
+Pillow==8.4.0
```
Comment (nickhib) [inline app/requirements.txt:5]: what is pillow being used for ?

**Review by nickhib — COMMENTED** [review]
just was looking over the pull request and wanted to clarify some things.

File: app/requirements.txt (hunk 2 above)
Comment (J26001) [reply app/requirements.txt:5]: Pillow allows to use ImageField in the model class to load images.

**Review by J26001 — APPROVED** [review]

**Pull request merged**